src/containers/Employees/index.js

```diff
@@ -39,8 +39,10 @@ export default class Employees extends React.Component {
   };
 
   removeUser = slackID => {
-    console.log(slackID);
-    this.setState({ selectedMojos: [] });
+    const { selectedMojos } = this.state;
+    const mojo = selectedMojos.findIndex(user => user.slackID === slackID);
+    selectedMojos.splice(mojo, 1);
```

Comment:
This is good, but we're mutating the `selectedMojos` array, which comes with some inherent risk. You could use `filter` instead, which returns a new array https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter